pkg/config/config.go

```diff
@@ -111,6 +111,7 @@ func LoadArgs(args ...string) (ProgArgs, error) {
 	flags.StringVar(&pArgs.RTE.KubeletConfigFile, "kubelet-config-file", "/podresources/config.yaml", "Kubelet config file path.")
 	flags.StringVar(&pArgs.RTE.PodResourcesSocketPath, "podresources-socket", "unix:///podresources/kubelet.sock", "Pod Resource Socket path to use.")
 	flags.BoolVar(&pArgs.RTE.PodReadinessEnable, "podreadiness", true, "Custom condition injection using Podreadiness.")
+	flags.BoolVar(&pArgs.RTE.OwnerReferenceEnable, "node-owns-nrt", false, "RTE will inject NRT's related node as OwnerReference to ensure cleanup if the node is deleted.")
```

Comment:
I'd rename the option like `add-nrt-owner`

Reply:
done